scripts/lib/upgrade-zulip-stage-2

```diff
@@ -123,7 +127,7 @@ if not args.skip_puppet or migrations_needed:
     # state.
     logging.info("Stopping Zulip...")
     subprocess.check_call(["supervisorctl", "stop", "zulip-workers:*", "zulip-django",
-                           "zulip-tornado", "zulip-senders:*"], preexec_fn=su_to_zulip)
+                           "zulip-tornado", "zulip-senders:*", "zulip-thumbor"], preexec_fn=su_to_zulip)
```

Comment:
This may cause problems on systems that don't have thumbor installed.  We should probably instead do something similar to what we do with `process-fts-updates` where we make this being in the list conditional.  

Since doing a second supervisord call for this is silly, we should probably just make the list of processes a variable, and include this or not depending on whether the `/etc/supervisord/conf.d` file exists.

(That same treatment doesn't make as much sense for `process-fts-updates`, since that one isn't part of the basically serving code path)